happiness-backend/api/journal.py

```diff
@@ -50,4 +51,42 @@ def get_entries(req, headers):
     page, count = req.get("page", 1), req.get("count", 10)
     # add password key to schema context so entries can be decrypted
     DecryptedJournalSchema.context['password_key'] = headers.get('password_key')
-    return journal_dao.get_entries_by_count(user.id, page, count)
\ No newline at end of file
+    return journal_dao.get_entries_by_count(user.id, page, count)
+
+@journal.put('/<int:id>')
```

Comment:
Why did you change the route for editing happiness to not include the ID, but the route for editing journals does? We should have consistency in the routes throughout our models.